ci/build_setup.sh

```diff
@@ -103,7 +103,6 @@ if [ "$1" != "-nofetch" ]; then
   # This is the hash on https://github.com/envoyproxy/envoy-filter-example.git we pin to.
   (cd "${ENVOY_FILTER_EXAMPLE_SRCDIR}" && git fetch origin && git checkout -f 5a05fb056a632a30b890ca85e69792558e90ffa9)
   sed -e "s|{ENVOY_SRCDIR}|${ENVOY_SRCDIR}|" "${ENVOY_SRCDIR}"/ci/WORKSPACE.filter.example > "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/WORKSPACE
-  cp -f "${ENVOY_SRCDIR}"/.bazelversion "${ENVOY_FILTER_EXAMPLE_SRCDIR}"/.bazelversion
```

Comment:
Will this break the version control we have over Bazel when we build from the filter example dir in CI?

Reply:
I'm having hard time parsing the sentence. The way I understand it is that filter-example symlinks bazel version to envoy git-submodule. So it's effectively the same bazel version here and there.

Reply:
You need this otherwise Bazelisk will always pull latest version, so it will break once newer version of bazel is released.

Reply:
Hmm, I wasn't aware of https://github.com/envoyproxy/envoy-filter-example/blob/master/.bazelversion is added, so this should be fine.